app/templates/Contributors.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% set currentPage = "contributors" %}

```

Comment:
Is this variable used? It's declared after base.html is included, and not used in your app_content block.